scripts/deploy/README.md

```diff
@@ -12,6 +12,7 @@ This document details how to deploy Chat Copilot's required resources to your Az
 - `F1` and `D1` SKUs for the App Service Plans are not currently supported for this deployment in order to support private networking.
 
 - Chat Copilot deployments use Azure Active Directory for authentication. All endpoints (except `/healthz` and `/authInfo`) require authentication to access.
+- This deployment only supports gpt-4O as chat completion model
```

Comment:
That's not true.  gpt-4 works and I _believe_ gpt-3.5-turbo, gpt-4-turbo and gpt-4-mini do too!

Which model(s) gave you problems?